src/homepage/homepage.html

```diff
@@ -45,14 +95,19 @@
     }
 
     #activities div:nth-child(2n + 1) {
-      background-color: var(--tertiary-color);
+      background-color: var(--primary-color);
     }
 
     #merchandise img {
       width: 450px;
       max-width: calc(100% - 20px);
     }
   </style>
+  <section id="logo">
+    <img src="../../images-optimized/logo.svg" alt="#Trending logo">
+    <span class="title--big">12TH LUSTRUM</span>
+    <span class="title--small">W.I.S.V. 'Christiaan Huygens'</span>
```

Comment:
Prefer to use semantic tags, preferably headers, instead of spans with styling. Makes it easier for screen readers to asess.